source/math/FloatVectorList.ooc

```diff
@@ -432,9 +433,51 @@ FloatVectorList: class extends VectorList<Float> {
 		}
 		(crossCorrelation, offsets)
 	}
+	// Assumes equi-distant samples
+	// https://en.wikipedia.org/wiki/Spline_interpolation
+	interpolate: func ~cubicSpline (numberOfPoints: Int) -> This {
```

Comment:
We usually don't use the ~ suffix in this way, where the caller have to explicitly write it out. Normally I would just call them interpolateLinear and interpolateSpline. Or maybe interpolate: func (numberOfPoints: Int, mode: InterpolationMode).